roles/pulp_api/templates/pulpcore-api.service.j2

```diff
@@ -17,7 +17,8 @@ RuntimeDirectory=pulpcore-api
 ExecStart={{ __pulp_daemons_dir }}/gunicorn pulpcore.app.wsgi:application \
           --bind '{{ pulp_api_bind }}' \
           --workers {{ pulp_api_workers }} \
-          --access-logfile -
+          --access-logfile - \
+          --access-logformat 'pulp [%({correlation-id}o)s]: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"'
```

Comment:
I am not sure if the "pulp ..." part makes sense. We do it [here](https://github.com/pulp/pulpcore/blob/4eafbdf63c3f574a25fe89aa4c653bbd6c73530c/pulpcore/app/settings.py#L189) though.

Reply:
Do you have an example of what the log line will look like?

Reply:
Here's what it looks like currently:

```
Nov 11 20:26:10 localhost gunicorn[27745]: 127.0.0.1 - admin [11/Nov/2020:20:26:10 +0000] "GET /pulp/api/v3/status/ HTTP/1.0" 200 861295 "-" "HTTPie/2.0.0"
```

And here's what it looks like for a request for a correlation id of `377880f1-efa3-4e2d-a500-a99a9bf370e2`:

```
Nov 11 20:31:42 localhost gunicorn[27745]: pulp [377880f1-efa3-4e2d-a500-a99a9bf370e2]: 127.0.0.1 - admin [11/Nov/2020:20:31:42 +0000] "GET /pulp/api/v3/status/ HTTP/1.0" 200 1242 "-" "HTTPie/2.0.0"
```

Reply:
That makes sense to me.